src/components/SliderComponent/SliderComponent.tsx

```diff
@@ -0,0 +1,91 @@
+'use client';
+import { SwiperClass } from 'swiper/react';
+import './scss/SliderComponent.scss';
+import 'swiper/scss';
+import 'swiper/scss/navigation';
+import React, { useEffect, useState } from 'react';
+import { Slider } from '../Slider/Slider';
+
+type SliderComponentProps = {
+  title: string;
+};
+
+export const SliderComponent: React.FC<SliderComponentProps> = ({ title }) => {
+  const [swiper, setSwiper] = useState<SwiperClass | null>(null);
+
+  const slidesAmount = {
+    desktop: 5,
+    tablet: 3,
+    mobile: 1,
+  };
+
+  const updateSlidesPerView = () => {
+    if (swiper && swiper.params) {
+      const width = window.innerWidth;
+
+      if (width >= 1300) {
+        swiper.params.slidesPerView = slidesAmount.desktop;
+      } else if (width >= 950) {
+        swiper.params.slidesPerView = slidesAmount.tablet;
+      } else {
+        swiper.params.slidesPerView = slidesAmount.mobile;
+      }
+
+      swiper.update();
+    }
+  };
+
+  useEffect(() => {
+    updateSlidesPerView();
+
+    window.addEventListener('resize', updateSlidesPerView);
+
+    return () => {
+      window.removeEventListener('resize', updateSlidesPerView);
+    };
+  }, [swiper]);
+
+  return (
+    <section className="section">
+      <div className="section__header">

```

Comment:
Consider using header tag instead div. Add class name to h2 tag.

Reply:
It's only the header of the component. I think it wouldn't be very semantic to use a header in every slider. And why that h2 need class ?

Reply:
Using classes effectively complements semantic HTML by providing a way to style and manage the presentation of your content without altering the core meaning conveyed by the elements themselves.

In essence, prioritize semantic elements for structure and meaning, and leverage classes for targeted styling and advanced control.